qutebrowser/completion/completer.py

```diff
@@ -249,7 +241,8 @@ def _quote(self, s):
         else:
             return s
 
-    def selection_changed(self, selected, _deselected):
+    @pyqtSlot(QItemSelection, QItemSelection)
+    def on_selection_changed(self, selected, _deselected):
```

Comment:
Since this is a slot now, you can drop the second argument (but in the slot and in the decorator) afaik (as slots should work fine with less arguments).